src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -257,6 +257,11 @@ protected boolean shouldProcessLine(ConcurStandardAccountingExtractDetailLine sa
                 return true;
             case ConcurConstants.PAYMENT_CODE_UNIVERSITY_BILLED_OR_PAID :
                 reportCorporateCardPayment(saeLine);
+                // Temporary hack to ignore personal expense lines until their handling has been implemented.

```

Comment:
Since this will be implemented right away, not going to push it either way.  However, I think notes like this are helpful.